app/models/publication_group.rb

```diff
@@ -6,7 +6,7 @@ class PublicationGroup < ApplicationRecord
 
   has_many :list_publication_groups, dependent: :destroy
 
-  has_many :vocab_distractors
+  has_many :vocab_distractors, foreign_key: :distractor_publication_group_id
```

Comment:
How was this working before? It would have been looking for `publication_group_id` which doesn't exist right?

Reply:
I think before it just wouldn't work if called, but because Rails 6.1 started caring about inverse relationships, this started actually crashing.